fronts-client/src/components/FrontsEdit/FrontSection/FrontSection.tsx

```diff
@@ -201,19 +215,36 @@ class FrontSection extends React.Component<
 						)}
 						<FrontHeaderMeta>
 							<EditModeVisibility visibleMode="fronts">
-								<a
-									href={`${
-										shouldUsePreviewCODE
-											? urls.previewUrlCODE
-											: urls.previewUrlPROD
-									}${this.props.frontId}`}
-									target="preview"
-								>
-									<FrontHeaderButton>
-										<PreviewEyeIcon size="xl" />
-										<PreviewButtonText>Preview</PreviewButtonText>
-									</FrontHeaderButton>
-								</a>
+								<LinkButtons>
+									<Link
+										href={`${
+											shouldUseCODELinks
+												? urls.previewUrlCODE
+												: urls.previewUrlPROD
+										}${this.props.frontId}`}
+										target="preview"
+									>
+										<FrontHeaderButton>
+											<PreviewEyeIcon size="xl" />
+											<LinkButtonText className="visible-based-on-front-header-width">
+												Preview
+											</LinkButtonText>
+										</FrontHeaderButton>
+									</Link>
+									<Link
+										href={`${
+											shouldUseCODELinks ? urls.liveUrlCODE : urls.liveUrlPROD
+										}${this.props.frontId}`}
+										target="live"
```

Comment:
I wonder if the `target` should be the name/path of the front? so that the tab can be re-used (as it is now) but can be different if you're working on multiple fronts at once